### PR TITLE
[DROOLS-5127] parse dmn resource only once during compilation

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNBuildFromReader.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import org.junit.Test;
+import org.kie.api.KieBase;
+import org.kie.api.KieServices;
+import org.kie.api.io.Resource;
+import org.kie.api.io.ResourceType;
+import org.kie.internal.utils.KieHelper;
+
+import static org.junit.Assert.assertNotNull;
+
+public class DMNBuildFromReader {
```
Comment (manstis) [inline kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNBuildFromReader.java]: Should this be named `.....Test` so SureFire detects it!?

Comment (mariofusco) [reply kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNBuildFromReader.java]: yes, sorry

Comment (mariofusco) [reply kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNBuildFromReader.java]: @manstis done, thanks for noticing

**Review by manstis — APPROVED** [review]
Approving (for what my review is worth!)

**Comment by tarilabs** [comment]
Please incorporate this https://github.com/mariofusco/drools/pull/12

**Pull request merged**